packages/cli/src/angular/migrations/standalone/0004-migrate-import-statements.ts

```diff
@@ -10,6 +10,10 @@ export const migrateImportStatements = async (
 ) => {
   // Get all typescript source files in the project and update any @ionic/angular imports to @ionic/angular/standalone
   for (const sourceFile of project.getSourceFiles()) {
+    if (sourceFile.getFilePath().endsWith(".html")) {
```

Comment:
Can we exclusively limit this to `.ts` extensions? While the intent is these should only be typescript files, we load additional file types into the project instance to be able to leverage the same API for migrations to HTML files in earlier migration steps. 

Thinking something like:
```ts
if (!sourceFile.getFilePath().endsWith(".ts")) {
  continue;
}
```

Reply:
Agree, made that change.